Formula/k6.rb

```diff
@@ -30,8 +30,7 @@ def install
   end
 
   test do
-    output = "Test finished"
-    assert_match output, shell_output("#{bin}/k6 run github.com/loadimpact/k6/samples/http_get.js 2>&1")
+    assert_match "whatever", shell_output("#{bin}/k6 run - 2>&1 <<< 'export default function() { console.log(`whatever`); }'")
```

Comment:
Instead of putting this on one line, you can write the line to a file and then just call the file in `shell_output`

Off the top of my head, something like this should work:

```ruby
(testpath/"whatever.js").write <<~EOS
  export default function() {
    console.log("whatever");
  }
EOS

assert_match "whatever", shell_output("#{bin}/k6 run whatever.js 2>&1")